Assets/HoloToolkit/Build/Editor/BuildDeployWindow.cs

```diff
@@ -124,8 +128,16 @@ private void OnGUI()
                     if (GUILayout.Button("Open SLN", GUILayout.Width(buttonWidth_Quarter)))
                     {
                         // Open SLN
-                        FileInfo slnFile = new FileInfo(Path.Combine(buildDirectory, PlayerSettings.productName + ".sln"));
-                        System.Diagnostics.Process.Start(slnFile.FullName);
+                        string slnFilename = Path.Combine(buildDirectory, PlayerSettings.productName + ".sln");
+                        if (File.Exists(slnFilename))
+                        {
+                            FileInfo slnFile = new FileInfo(slnFilename);
+                            System.Diagnostics.Process.Start(slnFile.FullName);
+                        }
+                        else
+                        {
+                            Debug.LogError("Solution file does not exist (" + slnFilename + ")");
```

Comment:
Maybe something like the below from the old BuildMenu? Might be more clear than an error in the logs.

` bool buildNow = EditorUtility.DisplayDialog("Solution Not Found", "We couldn't find the solution. Would you like to Build?", "Yes, Build", "No"); `